ow_plexil/src/plans/owlat_plans/OwlatTestSequence1.plp

```diff
@@ -0,0 +1,34 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// This Test Sequence is found in the User Guide under section 3.8.1
+// It tests ARM_MOVE_JOINTS, ARM_TARE_FS, ARM_MOVE_JOINTS_GUARDED
+
+
+#include "owlat-interface.h"
+
+OwlatTestSequence1:
+{
+
+  Real angles1[7] = #(0.0 -0.5 0.0 2.0 0.0 0.0 0.0);
+  Real angles2[7] = #(0.0 0.5 0.0 0.0 0.0 0.0 0.0);
+  Real angles3[7] = #(0.0 -0.5 0.0 0.0 0.0 0.0 0.0);
+  Real force_threshold = 5.0;
+  Real torque_threshold = 1.0;
+
+  // Get Arm into position above ground
+  LibraryCall OwlatArmMoveJoints(Relative=false, Angles=angles1);
+
+  // Tare the force sensor
+  LibraryCall OwlatArmTareFS();
+
+  // Move the arm until it comes in contact with the ground 
+  LibraryCall OwlatArmMoveJointsGuarded(Relative=true, Angles=angles2, Retracting=false,
+                                        ForceThreshold=force_threshold, TorqueThreshold=torque_threshold);
+
+  // Pull the arm back from the surface
+  LibraryCall OwlatArmMoveJointsGuarded(Relative=true, Angles=angles3, Retracting=true,
+                                        ForceThreshold=force_threshold, TorqueThreshold=torque_threshold);
```

Comment:
Would be more readable with shorter lines, e.g. last argument underneath the preceding one.